src/trading/tradingSdk.test.ts

```diff
@@ -74,18 +74,100 @@ describe('TradingSdk', () => {
     it('When logs option is set to true, then should display logs', async () => {
       const logSpy = jest.spyOn(console, 'log')
 
-      const sdk = new TradingSdk(
+      const sdk = new TradingSdk(defaultTrader, { enableLogging: true, orderBookApi })
+
+      await sdk.getQuote(defaultOrderParams)
+
+      expect(logSpy.mock.calls[0][0]).toContain('[COW TRADING SDK]')
+    })
+  })
+
+  describe('Quote and post limit order', () => {
+    it('Quote result order amounts should match the quote result', async () => {
+      // This value is calculated in suggestSlippageBps.ts
+      const smartSlippage = 56
+
+      const sdk = new TradingSdk(defaultTrader, { orderBookApi })
+
+      // We requested a quote for defaultOrderParams
+      const quote = await sdk.getQuote(defaultOrderParams)
+
+      // orderToSign sell amount is quote.sellAmount - quote.feeAmount
+      expect(+quote.quoteResults.orderToSign.sellAmount).toEqual(
+        Number(quoteResponseMock.quote.sellAmount) + Number(quoteResponseMock.quote.feeAmount),
+      )
+
+      expect(quote.quoteResults.suggestedSlippageBps).toBe(smartSlippage)
+
+      // 397760000000000000000
+      const expectedBuyAmountFromQuote = amountMinusPercent(
+        Number(quoteResponseMock.quote.buyAmount),
+        smartSlippage / 100,
+      )
+
+      // orderToSign buy amount is quote.buyAmount - slippage
+      expect(+quote.quoteResults.orderToSign.buyAmount).toEqual(expectedBuyAmountFromQuote)
+    })
+
+    it('The final order should take partnerFee and slippage into account', async () => {
+      const slippageBips = 70
+      const partnerFeeBps = 30
+      const smartSlippage = 56
```

Comment:
Why 2 different values for slippage?
Also, from the code, whatever is in the appData takes precedence.

Reply:
>Also, from the code, whatever is in the appData takes precedence.

Yes, since we use `sdk.postLimitOrder`, we ignore the suggested slippage and only set it before order signing using `appDataParams`